app/src/main/kotlin/com/wire/android/ui/common/bottomsheet/conversation/HomeSheetContent.kt

```diff
@@ -108,21 +110,55 @@ internal fun ConversationMainSheetContent(
                     )
                 }
             }
+
+            if (conversationSheetContent.canAddToFavourite() && !conversationSheetContent.isArchived) {
+                conversationSheetContent.isFavorite?.let { isFavorite ->
+                    if (isFavorite) {
+                        add {
+                            MenuBottomSheetItem(
+                                title = stringResource(R.string.label_remove_from_favourites),
+                                leading = {
+                                    MenuItemIcon(
+                                        id = R.drawable.ic_favourite,
+                                        contentDescription = stringResource(R.string.content_description_remove_from_favourites),
```

Comment:
please, use `contentDescription = null` if icon is not clickable and it's just duplicating title that goes after the icon. 
Otherwise Talkback will have a bit  broken behaviour ;P